ophyd/utils/epics_pvs.py

```diff
@@ -200,14 +217,17 @@ def waveform_to_string(value, type_=str, delim=''):
 
 @cached_retval
 def get_pv_form():
-    '''
-    Due to a bug in certain versions of PyEpics, form='time'
-    cannot be used with some large arrays.
+    '''Get the PV form that should be used for pyepics
+
+    Due to a bug in certain versions of PyEpics, form='time' cannot be used
+    with some large arrays.
 
     native: gives time.time() timestamps from this machine
     time: gives timestamps from the PVs themselves
 
-    :returns: 'native' or 'time'
+    Returns
+    -------
+    'native' or 'time'
```

Comment:
I would write this as `{'native', 'time'}`.  Only really important if you want to expose this to vistrails.